config/environments/development.rb

```diff
@@ -7,7 +7,7 @@
   config.cache_classes = false
 
   # Do not eager load code on boot.
-  config.eager_load = true
+  config.eager_load = false
```

Comment:
did you mean to leave this in?

Reply:
I did... do you know why we are doing eager loading? This eliminates that initial failure when we do code changes or the occasional need to restart the container...

Reply:
Hmm I'm a little concerned/unclear b/c the previous PR listed the change as "Also fixed an issue with rails console" https://github.com/chanzuckerberg/idseq-web/pull/1588